### DISCUSSION TITLE
Nominating @lolodomo as a maintainer

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-webui#1811] [WebUI Maintainer Discussions]
Dear @openhab/webui-maintainers,

I'd like to nominate @lolodomo for becoming a maintainer of this repo. He has been the main contributor to the Basic UI in the past year(s) and is clearly well into its codebase.
His role as a maintainer would therefore be specific to Basic UI and its PRs.

Please vote with 👍 or 👎  here on this post. Thanks!

**Comment by kaikreuzer** [comment]
Thanks for the quick votes and big welcome to @lolodomo!

**Comment by lolodomo** [comment]
Thank you guys.
